src/v3/src/util/theme.ts

```diff
@@ -10,319 +10,328 @@
+            // when rendering it as the startIcon for primary buttons
             ...(ownerState.variant === 'primary' && {
-              '&:hover': {
-                backgroundColor: t.palette.primary.dark,
-              },
-            }),
-            ...(ownerState.variant === 'secondary' && {
-              '&:hover': {
-                backgroundColor: t.palette.primary.lighter,
-                borderColor: t.palette.primary.light,
-                color: t.palette.primary.main,
+              '& .MuiCircularProgress-root': {
+                color: mergedTokens.HueNeutralWhite,
               },
             }),
-            ...(ownerState.variant === 'floating' && {
-              '&:hover': {
-                backgroundColor: 'rgba(29, 29, 33, 0.1)',
-                borderColor: 'transparent',
+            // Fix for IE11 - don't shrink icon if the button text is multiline
+            ...(ownerState.startIcon && {
+              '& .MuiButton-startIcon': {
+                flexShrink: 0,
               },
             }),
           }),
         },
       },
-      // ruleset with :focus-visible pseudo-selector break entire ruleset in
-      // ie11 because its not supported. re-define the :hover rule separately
-      // again so the ruleset is applied in ie11
-      MuiIconButton: {
+      MuiCheckbox: {
         styleOverrides: {
           root: {
-            '&:hover': {
-              backgroundColor: 'rgba(29, 29, 33, 0.1)',
-              borderColor: 'transparent',
+            // Odyssey uses gap for spacing between Checkbox and label which is not IE11-friendly
+            marginInlineEnd: mergedTokens.Spacing2,
+          },
+        },
+      },
+      MuiChip: {
+        styleOverrides: {
+          root: {
+            lineHeight: 'normal',
+          },
+        },
+      },
+      MuiFormControlLabel: {
+        styleOverrides: {
+          root: {
+            // Odyssey uses gap for spacing between Checkbox and label which is not IE11-friendly
+            gap: 0,
+          },
+          label: {
+            // Fixes text overflow in IE11
+            width: '100%',
+            // Forces Odyssey Checkbox hint text to be grey even during error state
+            '& .MuiFormHelperText-root.Mui-error': {
+              color: mergedTokens.TypographyColorSubordinate,
             },
           },
         },
       },
-
+      MuiFormHelperText: {
+        styleOverrides: {
+          root: {
+            display: 'block',
+          },
+        },
+      },
+      MuiInputBase: {
+        styleOverrides: {
+          root: ({ ownerState }) => ({
+            width: '100%',
+            // Odyssey sets flex: "1" but that results in the following IE11 flexbug
+            // https://github.com/philipwalton/flexbugs?tab=readme-ov-file#flexbug-7
+            flex: 'auto',
+            ...(ownerState.name && isLtrField(ownerState.name) && {
+              direction: 'ltr',
+            }),
```

Comment:
Forcing specific input components to be LTR by default at the theme override level